install_script.sh

```diff
@@ -3,30 +3,60 @@ mkdir  -p $1.dir
 OLD_PWD=$pwd
 
 # Extract
-#tar -xvf $1 -C $1.dir/
+#tar --lzma -xvf $1 -C $1.dir/
 cd $1.dir
 ar x ../$1
-tar -xvf data.tar.*
+tar --lzma -xvf data.tar.*
```

Comment:
Change it to:
if [[ $(ls | grep data.tar.lzma ) ]]; then
tar --lzma -xvf data.tar.lzma
else
tar -xvf data.tar.*
fi

I think there will never be more than one data.tar.* file.